### ISSUE TITLE
训练时遇到Aborted at 1516977979 (unix time) try "date -d @1516977979" if you are using GNU date *** PC: @                0x0 (unknown) *** SIGFPE (@0x7ff24e3acb3e) received by PID 2262 (TID 0x7ff27157d740) from PID 131247596

### ISSUE THREAD

**Issue by HljWafer** [PaddlePaddle/Paddle#7949]
训练时遇到报错，求老师们给点思路。
Aborted at 1516977979 (unix time) try "date -d @1516977979" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGFPE (@0x7ff24e3acb3e) received by PID 2262 (TID 0x7ff27157d740) from PID 1312475966; stack trace: ***
    @     0x7ff271170150 (unknown)
    @     0x7ff24e3acb3e paddle::AucEvaluator::evalImp()
    @     0x7ff24e3ac586 paddle::Evaluator::eval()
    @     0x7ff24e397868 paddle::CombinedEvaluator::eval()
    @     0x7ff24e1efbd6 _wrap_GradientMachine_eval
    @     0x55680e42899e PyEval_EvalFrameEx
    @     0x55680e41fb3a PyEval_EvalCodeEx
    @     0x55680e427e1f PyEval_EvalFrameEx
    @     0x55680e41fb3a PyEval_EvalCodeEx
    @     0x55680e42782e PyEval_EvalFrameEx
    @     0x55680e41fb3a PyEval_EvalCodeEx
    @     0x55680e45050f (unknown)
    @     0x55680e44b202 PyRun_FileExFlags
    @     0x55680e44acad PyRun_SimpleFileExFlags
    @     0x55680e3f9d7b Py_Main
    @     0x7ff270d9e1c1 __libc_start_main
    @     0x55680e3f95fa _start
    @                0x0 (unknown)

最开始一启动训练就报同样的错 是数据的问题，因为数据有空行
改了一部分代码后，每次在训练一段时间后（大概30分钟）会报这个错，
代码是官网中栈式双向LSTM改的，主要的变化就是输入从单层数据改成了多层数据

    assert stacked_num % 2 == 1
    fc_para_attr = paddle.attr.Param(learning_rate=8e-4)
    lstm_para_attr = paddle.attr.Param(initial_std=0.3, learning_rate=0.003)
    para_attr = [fc_para_attr, lstm_para_attr]
    bias_attr = paddle.attr.Param(initial_std=0.001, l2_rate=0.001)

    relu = paddle.activation.Relu()
    linear = paddle.activation.Linear()

    data = paddle.layer.data(
        "word", paddle.data_type.integer_value_sub_sequence(dict_dim))
    emb_group = embedding_layer(input=data, size=conf.emb_size)

    def lstm_group(lstm_group_input):
        fc1 = paddle.layer.fc(input=lstm_group_input,
                              size=conf.hidden_size,
                              act=linear,
                              bias_attr=bias_attr)
        return fc1

    lstm_nest_group = recurrent_group(
        input=SubsequenceInput(emb_group), step=lstm_group, name="lstm_nest_group")

    lstm1 = paddle.layer.lstmemory(
        input=lstm_nest_group, act=relu, bias_attr=bias_attr)

    inputs = [lstm_nest_group, lstm1]
    for i in range(2, stacked_num + 1):
        fc = paddle.layer.fc(input=inputs,
                             size=conf.hidden_size,
                             act=linear,
                             param_attr=para_attr,
                             bias_attr=bias_attr)
        lstm = paddle.layer.lstmemory(
            input=fc,
            reverse=(i % 2) == 0,
            act=relu,
            bias_attr=bias_attr)
        inputs = [fc, lstm]

    fc_last = paddle.layer.pooling(input=inputs[0], pooling_type=paddle.pooling.Max())
    lstm_last = paddle.layer.pooling(input=inputs[1], pooling_type=paddle.pooling.Max())
    output = paddle.layer.fc(input=[fc_last, lstm_last],
                             size=class_dim,
                             act=paddle.activation.Softmax(),
                             bias_attr=bias_attr,
                             param_attr=para_attr)

    if not is_predict:
        lbl = paddle.layer.data("label", paddle.data_type.integer_value(2))
        cost = paddle.layer.classification_cost(input=output, label=lbl)
        return cost, output, lbl
    else:
        return output

目前测试的话， 如果是单层输入的话 ， 是不报错的， 但是多层输入会报错。
请各位老师给点思路， 是模型配置的不对还是数据有问题。错误栈信息每次都是AucEvaluator相关，的是在做评估时出了问题吗？

**Comment by kuke** [comment]
@HljWafer 请问能确认现在的数据没有问题了吗？另外检查一下数据和连接的维度，这种也比较容易出问题。

**Comment by yeyupiaoling** [comment]
@HljWafer 
这个是浮点异常吧? 你试试减少学习率看看, 是否有效
